forte/utils/payload_factory.py

```diff
@@ -0,0 +1,159 @@
+from abc import abstractmethod
+from multiprocessing.sharedctypes import Value
+from typing import Callable
+import numpy as np
+from forte.data.ontology.top import Meta
+
+
+class PayloadFactory:
+    """
+    A class that handles the creation of payloads. It can register meta data and check its validity.
+    """
+
+    def __init__(self):
+        self.valid_meta = (
+            {}
+        )  # map from meta data name to function that handles this meta data
+
+    def register(self, meta: Meta):
+        """
+        A function that registers a meta data type into the factory.
+
+        Args:
+            meta_name: a Generic object that is used to register a Payload meta data type.
+        """
+        if meta.source_type not in ("web", "local"):
+            raise ValueError("Meta data source must be either 'web' or 'local'")
+        self.valid_meta[type(meta)] = True
+
+    def check_meta(self, meta):
+        return type(meta) in self.valid_meta
+
+
+class Payloading:
```

Comment:
Why are we creating so many classes? The `PayloadFactory` only stores a dictionary, and then this class only stores an empty abstract method. Can we just do it with one class?

Reply:
I was referring to the template that you wrote https://slack-files.com/T222W2A87-F03KJLMM293-a88d5aa6a3 and tried to follow it. And think I can make up the rest function after the skeleton is built up.

Reply:
Ok, so we won't merge any unfinished structures. I am not sure if these belong to this PR.